src/stats/manager_impl.go

```diff
@@ -0,0 +1,119 @@
+package stats
+
+import (
+	pb_struct "github.com/envoyproxy/go-control-plane/envoy/extensions/common/ratelimit/v3"
+	"github.com/envoyproxy/ratelimit/src/settings"
+	gostats "github.com/lyft/gostats"
+	logger "github.com/sirupsen/logrus"
+)
+
+func NewStatManager(store gostats.Store, s settings.Settings) *ManagerImpl {
```

Comment:
renaming `s` settings would probably clash with `settings` package, but it could have improved readability